src/routes/jira/jira-get-connections-backfillStatus.ts

```diff
@@ -0,0 +1,150 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
the name of the file doesn't quite follow the pattern that we use, it should be `jira-get-connections-backfill-status.ts` (as well as the test one)

Reply:
corrected